### ISSUE TITLE
MD064 with allow-sentence-double-space doesn't understand some cases

### ISSUE THREAD

**Issue by federicomenaquintero** [rvben/rumdl#364]
### rumdl version

0.1.7

### Installation method

cargo install

### Bug description

Rule MD064, even when `allow-sentence-double-space` is turned on, produces warnings for sentences that end in markup before the period.

### Steps to reproduce

1. Create `foo.md` with this:

```markdown
<!-- rumdl-configure-file { "MD064": { "allow-sentence-double-space": true } } -->

# Title

what the font size is for the toplevel `<svg>`.  Fortunately, librsvg

And here is where I want to say, SVG documents are **scalable**.  Pick

That's right, no `width`, no `height`, no `viewBox`.  There is no easy

**SVG documents are scalable**.  That's their whole reason for being!
```

(These lines are extracted from librsvg's documentation.)

2. Run `rumdl check foo.md`

### Expected behavior

No warnings.

### Actual behavior

```
$ rumdl check ~/src/foo.md 
/home/federico/src/foo.md:5:48: [MD064] Multiple consecutive spaces (2) found [*]
/home/federico/src/foo.md:7:65: [MD064] Multiple consecutive spaces (2) found [*]
/home/federico/src/foo.md:9:53: [MD064] Multiple consecutive spaces (2) found [*]
/home/federico/src/foo.md:11:32: [MD064] Multiple consecutive spaces (2) found [*]
```

### Configuration

```toml

```

### Operating system

Linux

**Comment by federicomenaquintero** [comment]
Another line like that!

```markdown
**You should not just use rsvg-convert to render test files!**  The
```

**Comment by rvben** [comment]
Fixed in v0.1.8. Sentences ending with markup (like `` `<svg>`.  `` and `**scalable**.  `) are now correctly recognized when `allow-sentence-double-space` is enabled.

**Comment by rvben** [comment]
Thanks for the bug report @federicomenaquintero! Closing as fixed in v0.1.8.

**Comment by federicomenaquintero** [comment]
Thanks for the quick fix!
